taipy/gui/_renderers/factory.py

```diff
@@ -597,6 +597,19 @@ class _Factory:
             ]
         )
         ._set_propagate(),
+        "ProgressBar": lambda gui, control_type, attrs: _Builder(
```

Comment:
I suppose `Progress` is more generic than `ProgressBar` (especially when we want to support circular progress)

Reply:
I'm renaming it to `Progress` instead of `ProgressBar`👍🏼